src/wp-admin/edit-form-advanced.php

```diff
@@ -462,12 +462,17 @@
 		)
 	);
 endif;
+
+$connection_lost_message = '<span class="spinner"></span> ' . __( '<strong>Connection lost.</strong> Saving has been disabled until you are reconnected.' ) . '<span class="hide-if-no-sessionstorage">' . __( 'This post is being backed up in your browser, just in case.' ) . '</span>';
```

Comment:
Wondering if we can split this over multiple lines to save on horizontal scrolling?

Reply:
It doesn't really have great places for breaking.  Nothing that really won my heart, anyway... I considered breaking it into two messages: `$saving_disabled_message` and `$backed_up_message`, then appending those together for the message. Not thrilled with that option, but it's the best I can come up with at the moment.

Reply:
I'm hesitant to add a `sprintf()`, but it is used in many places for the notices already.

Something like:
```suggestion
$connection_lost_message = sprintf(
	'<span class="spinner"></span> %1$s<span class="hide-if-no-sessionstorage">%2$s</span>',
	__( '<strong>Connection lost.</strong> Saving has been disabled until you are reconnected.' ),
	__( 'This post is being backed up in your browser, just in case.' )
);
```

Reply:
That's definitely an option. And, as you say, it's a pattern that is pretty well represented in similar contexts.

Reply:
I'll leave it to your discretion - It's certainly not a blocker.